src/main/scala/co/teapot/tempest/graph/MemMappedDynamicDirectedGraphConverter.scala

```diff
@@ -42,33 +42,33 @@ object MemMappedDynamicDirectedGraphConverter {
               outputGraphFile: File,
               log: String => Unit = System.err.println): Unit = {
     val graph = new MemMappedDynamicDirectedGraph(outputGraphFile, syncAllWrites=false)
-    val outDegrees = new IntBigArrayBigList()
-    val inDegrees = new IntBigArrayBigList()
+    val outDegrees = new IntArrayList()
+    val inDegrees = new IntArrayList()
 
     Util.logWithRunningTime(log, "first pass", printAtStart = true) {
-      FileUtil.forEachLongPair(edgeListFile, log, linesPerMessage = 1000000) { (id1, id2) =>
-        if (id1 >= outDegrees.size64) {
-          outDegrees.size(id1 + 1) // Note that size fills new entries with 0s
+      FileUtil.forEachIntPair(edgeListFile, log, linesPerMessage = 1000000) { (id1, id2) =>
+        if (id1 >= outDegrees.size) {
+          outDegrees.addAll(IntArrayList.wrap(new Array[Int](id1 + 1 - outDegrees.size)))
```

Comment:
I'm not sure I follow here. When `id1 = outDegrees.size` you get `outDegrees.size + 1 - outDegrees.size = 1` so it seems like you're growing by one item, no?